pkg/sfs/controllerserver.go

```diff
@@ -39,47 +44,62 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	// check pv hasCreate or not
+	if value, ok := pvcProcessSuccess[req.Name]; ok && value != nil {
```

Comment:
Checking for existence with `pvcProcessSuccess` may caust bugs,
it is better to call the service to check for existence.

Reply:
Modifications have been completed